src/test/renderer.test.cpp

```diff
@@ -52,6 +52,14 @@ class TextureObject : public scene::GameObject {
     bool m_keepAspectRatio;
 };
 
+// Global inputOption for visibility in other functions.
+std::shared_ptr<menu::InputOption> inputOption =
+    std::make_shared<menu::InputOption>("inputOption", 1.0);
+
+void HandleEscapeMenuInput(void *, events::KeyPressEventArgs &args) {
+    inputOption->HandleKeyPressEvent(args);
+}
```

Comment:
This should not be needed, the `inputOption::HandleKeyPressEvent` method can be called directly from the event handler using `BIND_EVENT_HANDLER_FROM(menu::InputOption::HandleKeyPressEvent, inputOption)`. You would just need to add the sender argument to that method.